ViewControllers/Search/SearchLogicController.swift

```diff
@@ -11,11 +11,13 @@ protocol SearchLogicControllerProtocol: class {
     
     var view: SearchViewControllerProtocol? { get set }
     func getArtists(name: String)
+    func resetArtistList()
 }
 
 final class SearchLogicController: SearchLogicControllerProtocol {
     
     weak var view: SearchViewControllerProtocol?
+    var emptyArtistList: [ArtistModel] = []
```

Comment:
looks little bi strange, if it is supposed to be empty, why it is var? and empty array doesn't need a variable, I would say...